### PR TITLE
[MAINTENANCE] add --ignore-suppress and --ignore-only-for to build_gallery.py with bugfixes

### PR DESCRIPTION
**Pull request by kenwade4**
Changes proposed in this pull request:
- Add 'dev' key to extras_require with all requirements
- Add ignore_suppress and ignore_only_for kwargs to choose to test more things sometimes
- Add requirements-dev-all-contrib-expectations.txt file
- Add protobuf to constraints-dev.txt
- Use correct import for contrib/cli package
- Use correct class name for some contrib Expectations and use correct package name
- Update convert_to_json_serializable to handle LineString obj

### CODE REVIEW

File: assets/scripts/build_gallery.py [hunk 1]
```diff
@@ -535,6 +539,22 @@ def format_docstring_to_markdown(docstr: str) -> str:
...
+    "--ignore-suppress",
+    "-c",
+    "ignore_suppress",
+    is_flag=True,
+    default=False,
+    help="Ignore the suppress_test_for list on Expectation sample tests",
+)
+@click.option(
+    "--ignore-only-for",
+    "-c",
```
Comment (rdodev) [inline assets/scripts/build_gallery.py]: @kenwade4 same short-hand flag as above?

File: contrib/experimental/great_expectations_experimental/expectations/expect_column_minimum_bounding_radius_to_be_between.py [hunk 2]
```diff
@@ -299,4 +299,4 @@ def _validate(
 
 
 if __name__ == "__main__":
-    ExpectColumnMininumBoundingRadiusToBeBetween().print_diagnostic_checklist()
+    ExpectColumnMinimumBoundingRadiusToBeBetween().print_diagnostic_checklist()
```
Comment (rdodev) [inline contrib/experimental/great_expectations_experimental/expectations/expect_column_minimum_bounding_radius_to_be_between.py:302]: not seeing what the delta is here. They look identical.

Comment (kenwade4) [reply contrib/experimental/great_expectations_experimental/expectations/expect_column_minimum_bounding_radius_to_be_between.py:302]: numBound vs mumBound

**Review by rdodev — APPROVED** [review]
LGTM

**Pull request merged**